### DISCUSSION TITLE
TrayIcon NativeMenuItem no pointerover feedback

### DISCUSSION THREAD

**Discussion by tkefauver** [AvaloniaUI/Avalonia#8744] [Q&A] [answered]
Hi, I'm running 0.10.16 and built my system tray icon like the `ControlCatalog.Desktop` sample but for some reason when the tray menu is shown there is no pointerover style changing (ie. the background of the menu item doesn't change) in MY project and I must be doing something wrong because in the sample everything works as expected! 

I'm currently testing in Windows 10 and have tried using both default themes in my `App.axaml` `Application.Styles`.

For Fluent I use:
`<FluentTheme Mode="Light" />`

And Simple:
`<StyleInclude Source="avares://Avalonia.Themes.Default/DefaultTheme.xaml" />
   StyleInclude Source="avares://Avalonia.Themes.Default/Accents/BaseLight.xaml" />`

And my `TrayIcons` definition in `App.axaml` is exactly like `ControlCatalog.App.axaml`:

```
<TrayIcon.Icons>
    <TrayIcons>
      <TrayIcon Icon="/Assets/test_icon.ico" ToolTipText="Avalonia Tray Icon ToolTip">
        <TrayIcon.Menu>
          <NativeMenu>
            <NativeMenuItem Header="Settings">
              <NativeMenu>
                <NativeMenuItem Header="Option 1" ToggleType="Radio" IsChecked="True" Command="{Binding ToggleCommand}" />
                <NativeMenuItem Header="Option 2" ToggleType="Radio" IsChecked="True" Command="{Binding ToggleCommand}" />
                <NativeMenuItemSeparator />
                <NativeMenuItem Header="Option 3" ToggleType="CheckBox" IsChecked="True" Command="{Binding ToggleCommand}" />
                <NativeMenuItem Icon="/Assets/test_icon.ico" Header="Restore Defaults" Command="{Binding ToggleCommand}" />
                <NativeMenuItem Header="Disabled option" IsEnabled="False" />
              </NativeMenu>
            </NativeMenuItem>
            <NativeMenuItem Header="Exit" Command="{Binding ExitCommand}" />
          </NativeMenu>
        </TrayIcon.Menu>
      </TrayIcon>
    </TrayIcons>
  </TrayIcon.Icons>
```

Any ideas what I maybe doing wrong?

**Comment by maxkatz6** [comment, marked as answer]
It was fixed in 0.10.17

**Reply by tkefauver** [reply]
Ok, great it does work thank you!

**Reply by tkefauver** [reply]
@maxkatz6 This problem is back again in 0.10.18 running on Windows 11 in my application... The ControlCatalog sample works fine though so I'm not sure what happened. Any ideas what I maybe doing wrong?

**Reply by tkefauver** [reply]
@maxkatz6 Finally realized what was wrong... I had all `SystemControlHighlightList*` `SolidColorBrush`'s set to `Transparent` omg finally :)
